darts/models/regression_ensemble_model.py

```diff
@@ -58,25 +58,87 @@ def __init__(self,
         self.regression_model = regression_model
         self.train_n_points = regression_train_n_points
 
-    def fit(self, series: TimeSeries) -> None:
-        super().fit(series)
+    def fit(self,
+            series: Union[TimeSeries, Sequence[TimeSeries]],
+            past_covariates: Optional[Union[TimeSeries, Sequence[TimeSeries]]] = None,
+            future_covariates: Optional[Union[TimeSeries, Sequence[TimeSeries]]] = None
+            ) -> None:
+
+        super().fit(series, past_covariates=past_covariates, future_covariates=future_covariates)
 
         # spare train_n_points points to serve as regression target
-        raise_if(len(self.training_series) <= self.train_n_points,
+        if self.is_univariate:
```

Comment:
but then I will have to turn it back from `[series]` to `series` at the end, right? Note: currently there are far many checks because the code simplified from other reasons. Let me know if you still want me to work on it.

Reply:
Yes at the very end you'll need a check like 
```
return results[0] if self._is_single_series else results
```
But I think the way it is now is already OK.